README.markdown

```diff
@@ -527,6 +527,10 @@ var diameter: Double {
 }
 ```
 
+### Property Observers
+
+When writing didSets make sure that the logic is no more than a couple of lines long. If they need to be longer write the logic inside a private method or inside an external component.
```

Comment:
I'd say that it could be acceptable to reach a few lines (as sometimes you are setting some setup data in some component, e.g. a tableview where you set the delegate, datasource and probably a couple of other ui configs (background) etc. So, in my opinion, a few lines if they are related to setting up a component and they don't encapsulate logic, would be acceptable...